### PR TITLE
Updated: script and style file

### PR DESCRIPTION
**Pull request by ymaheshwari1**
- Updated: solr-query to only fetch the retail store information
- Improved z-index of bopis modal and overlay

### CODE REVIEW

File: scripts/shopify-tag.css [hunk 1]
```diff
@@ -74,12 +74,12 @@ p {
     top: 0;
     left: 0;
     opacity: 50%;
-    z-index: 100;
+    z-index: 2000;
     backdrop-filter: brightness(0.5);
 }
 
-#hc-store-information {
```
Comment (dt2patel) [inline scripts/shopify-tag.css:81]: what was max height doing before and why is it ok to remove it now?

Comment (ymaheshwari1) [reply scripts/shopify-tag.css:81]: We are using it to have a fixed height of store-info div, previously it does not have any effect as we have declared `hc-store-information` as a class but using it as id, so the max-height was not even applying, so removed it and used class selector.

Comment (dt2patel) [reply scripts/shopify-tag.css:81]: got it

File: scripts/script-tag.js [hunk 2]
```diff
@@ -142,7 +142,8 @@
             })
 
         } else if(location.pathname.includes('cart')) {
-            jQueryBopis("[data-cart-item-property-name]:contains('pickupstore')").closest('li').hide();
+            // jQueryBopis("[data-cart-item-property-name]:contains('pickupstore')").closest('li').hide();
+            jQueryBopis('.cart__text div:contains("pickupstore")').hide();
```
Comment (adityasharma7) [inline scripts/script-tag.js]: Is it generic?

**Review by adityasharma7 — CHANGES_REQUESTED** [review]

File: scripts/script-tag.js (hunk 2 above)
Comment (ymaheshwari1) [reply scripts/script-tag.js]: No, its specific to this store.

Comment (ymaheshwari1) [reply scripts/script-tag.js]: I will explore to find a generic way, so we don't need to update this code everytime the theme changes.

Comment (ymaheshwari1) [reply scripts/script-tag.js]: Reverted the change.

**Review by adityasharma7 — APPROVED** [review]

**Pull request merged**